### DISCUSSION TITLE
Exclude service accounts from brute force detection?

### DISCUSSION THREAD

**Discussion by Guite** [keycloak/keycloak#25535] [Q&A]
Hello everybody,

I want to ask whether it is possible that a service account would not be affected by brute force detection. Background: if three system instances use the same client but one of them is not configured correctly (e.g. using a wrong secret) then this should not stop the client from working properly so the other two instances will not be affected.

Thanks in advance for any pointers.

**Comment by andyhinds** [comment]
Hi @Guite did you get anywhere with this? I have the same question. In my case I want to run synthetic/smoke tests in production which requires lots of logins from various runners/IP addresses... thanks!

**Reply by Guite** [reply]
I have not pursued the matter further to date.
